src/EdFi.AnalyticsMiddleTier.Tests/Dimensions/When_querying_the_StaffSectionDim_view.cs

```diff
@@ -462,5 +462,22 @@ public void Then_should_have_YearsOfPriorTeachingExperience_Zero()
                 testResult.success.ShouldBe(true, testResult.errorMessage);
             }
         }
+
+        public class Given_StaffSection_142990_NCMR11
+       : When_querying_the_StaffSectionDim_view
+        {
+            public Given_StaffSection_142990_NCMR11(TestHarnessBase dataStandard) => SetDataStandard(dataStandard);
+
+            private const string _caseIdentifier = "142990_NCMR11";
+
+            [Test]
+            public void Then_should_have_HispanicLatinoEthnicity_Zero()
+            {
+                (bool success, string errorMessage) testResult = DataStandard.RunTestCase<CountResult>($"{TestCasesFolder}.{DataStandard.DataStandardBaseVersion}.{_caseIdentifier}_should_have_HispanicLatinoEthnicity_Zero.xml");
```

Comment:
The test will work when you change the result type from `CountResult` to `StaffSectionDim`.
DS 2 still doesn't work. Doesn't work I mean that when I change the second parameter of the COALESCE function to something different to 0, no tests fail.